modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/translator/TranslatorRegistryImpl.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.internal.translator;
+
+import com.liferay.osgi.service.tracker.collections.list.ServiceTrackerList;
+import com.liferay.osgi.service.tracker.collections.list.ServiceTrackerListFactory;
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+import com.liferay.translation.translator.Translator;
+import com.liferay.translation.translator.TranslatorRegistry;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+
+/**
+ * @author Alicia Garcia
+ */
+@Component(service = TranslatorRegistry.class)
+public class TranslatorRegistryImpl implements TranslatorRegistry {
+
+	@Override
+	public Translator getFirstTranslator(long companyId)
+		throws ConfigurationException {
+
+		Stream<Translator> stream = getTranslators(
+			companyId
+		).stream();
+
+		return stream.findFirst(
+		).orElse(
+			null
+		);
```

Comment:
You cannot chain the call to `stream()`; you need to introduce a variable for the result of `getTranslators()`.

Also, in this case, streams may be a bit overkill. Why not something like:
```
List<Translator> translators = ...;

if (translators.isEmpty()) {
    return null;
}

return translators.get(0);
```

Reply:
K.I.S. (s)